api/makefile

```diff
@@ -11,7 +11,7 @@ install-db:
 	docker run --name comfy-db -p 5432:5432 -d postgres:9.6
 	sleep 5s
 	psql -c "CREATE DATABASE ${DATABASE}"
-	psql ${DATABASE} < var/schema.sql
+	./node_modules/db-migrate/bin/db-migrate up --config ./database.js
```

Comment:
Don't you need to add the environment ? `db-migrate up --config ./database.js -e dev` or `-e production` ?

Or even better, in `database.js`, just require the needed configuration thanks to the NODE_ENV.